python/pylibraft/pylibraft/neighbors/__init__.py

```diff
@@ -12,6 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
+# from .brute_force import knn as brute_force_knn
```

Comment:
the brute force changes diff seem to be coming from PR #1292, not sure if they need to be included here or it was accidental?